modules/hosted-control-planes-troubleshooting.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-managing.adoc
+
+:_content-type: PROCEDURE
+[id="hosted-control-planes-troubleshooting_{context}"]
+= Troubleshooting hosted control planes
+
+When you need to troubleshoot an issue with hosted control plane clusters, you can gather information by running the `hypershift dump cluster` command. The command generates output for the management cluster and the hosted cluster.
+
+The output for the management cluster contains the following content:
+
+* Cluster-scoped resources: These resources are node definitions of the management cluster.
```

Comment:
The item kind of gets lost with the description. I'd consider playing with some other ways to present it to make it more scannable. For example, either by putting them into a table, or even just bolding the item. Like:

* **Cluster-scoped resources**: These resources are node definitions of the management cluster.